2024-09-18 14:47:14,453 INFO    MainThread:1664834 [wandb_setup.py:_flush():77] Current SDK version is 0.17.8
2024-09-18 14:47:14,454 INFO    MainThread:1664834 [wandb_setup.py:_flush():77] Configure stats pid to 1664834
2024-09-18 14:47:14,454 INFO    MainThread:1664834 [wandb_setup.py:_flush():77] Loading settings from /home/geo_llm/.config/wandb/settings
2024-09-18 14:47:14,454 INFO    MainThread:1664834 [wandb_setup.py:_flush():77] Loading settings from /home/geo_llm/llama3-finetune/wandb/settings
2024-09-18 14:47:14,454 INFO    MainThread:1664834 [wandb_setup.py:_flush():77] Loading settings from environment variables: {}
2024-09-18 14:47:14,454 INFO    MainThread:1664834 [wandb_setup.py:_flush():77] Applying setup settings: {'_disable_service': False}
2024-09-18 14:47:14,454 INFO    MainThread:1664834 [wandb_setup.py:_flush():77] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program_abspath': '/home/geo_llm/llama3-finetune/main.py', 'program': '/home/geo_llm/llama3-finetune/main.py'}
2024-09-18 14:47:14,454 INFO    MainThread:1664834 [wandb_setup.py:_flush():77] Applying login settings: {}
2024-09-18 14:47:14,454 INFO    MainThread:1664834 [wandb_init.py:_log_setup():524] Logging user logs to /home/geo_llm/llama3-finetune/wandb/run-20240918_144714-gjuyjnw5/logs/debug.log
2024-09-18 14:47:14,455 INFO    MainThread:1664834 [wandb_init.py:_log_setup():525] Logging internal logs to /home/geo_llm/llama3-finetune/wandb/run-20240918_144714-gjuyjnw5/logs/debug-internal.log
2024-09-18 14:47:14,455 INFO    MainThread:1664834 [wandb_init.py:init():607] calling init triggers
2024-09-18 14:47:14,455 INFO    MainThread:1664834 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {}
2024-09-18 14:47:14,455 INFO    MainThread:1664834 [wandb_init.py:init():657] starting backend
2024-09-18 14:47:14,455 INFO    MainThread:1664834 [wandb_init.py:init():661] setting up manager
2024-09-18 14:47:14,458 INFO    MainThread:1664834 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-09-18 14:47:14,460 INFO    MainThread:1664834 [wandb_init.py:init():669] backend started and connected
2024-09-18 14:47:14,462 INFO    MainThread:1664834 [wandb_init.py:init():767] updated telemetry
2024-09-18 14:47:14,468 INFO    MainThread:1664834 [wandb_init.py:init():800] communicating run to backend with 90.0 second timeout
2024-09-18 14:47:14,881 INFO    MainThread:1664834 [wandb_init.py:init():851] starting run threads in backend
2024-09-18 14:47:18,444 INFO    MainThread:1664834 [wandb_run.py:_console_start():2463] atexit reg
2024-09-18 14:47:18,445 INFO    MainThread:1664834 [wandb_run.py:_redirect():2309] redirect: wrap_raw
2024-09-18 14:47:18,445 INFO    MainThread:1664834 [wandb_run.py:_redirect():2374] Wrapping output streams.
2024-09-18 14:47:18,445 INFO    MainThread:1664834 [wandb_run.py:_redirect():2399] Redirects installed.
2024-09-18 14:47:18,447 INFO    MainThread:1664834 [wandb_init.py:init():894] run started, returning control to user process
2024-09-18 14:47:28,034 WARNING MsgRouterThr:1664834 [router.py:message_loop():77] message_loop has been closed
